chia/wallet/nft_wallet/nft_wallet.py

```diff
@@ -357,7 +356,7 @@ async def generate_new_nft(
         coins = await self.standard_wallet.select_coins(amount)
         if coins is None:
             return None
-        self.log.debug("Attempt to generate a new NFT")
+        self.log.debug("Attempt to generate a new NFT to %s", target_puzzle_hash)
```

Comment:
We probably don't want the representation here.

```suggestion
        self.log.debug("Attempt to generate a new NFT to: %s", target_puzzle_hash.hex())
```